src/connecpy/exceptions.py

```diff
@@ -10,7 +10,7 @@ class ConnecpyException(Exception):
     pass
 
 
-class ConnecpyServerException(httplib.HTTPException):
+class ConnecpyServerException(ConnecpyException):
```

Comment:
This seems to allow using `ConnecpyException` to "propagate exceptions to user without wrapping"